docs/docs/spacetimeauth/index.md

```diff
@@ -0,0 +1,95 @@
+# SpacetimeAuth - Overview
+
+> ⚠️ SpacetimeAuth is currently in beta, some features may not be available yet or may change in the future. You might encounter bugs or issues while using the service. Please report any problems you encounter to help us improve SpacetimeAuth.
+
+SpacetimeAuth is a service for managing authentication for your SpacetimeDB
+applications. This allows you to authenticate users without needing
+an external authentication service or even a hosting server.
+SpacetimeAuth is an OpenID Connect (OIDC) provider, which means it can be used with
+any OIDC-compatible client library.
+
+At the end of the authentication flow, your application receives an ID token
+containing identity claims (such as email, username, and roles). This token can
+then be used with any SpacetimeDB SDK to authenticate and authorize users.
+
+## Features
+
+### Authentication methods
+
+- Magic link
+- Github
+- Google
+- Discord
+- Twitch
+- Kick
+- (Coming soon) Steam: authenticate users via Steam and verify game or DLC ownership
```

Comment:
I'd consider removing this. @gefjon wrote a style guide which requires that we don't reference upcoming features in the docs, and there is an element of good hygiene to that.

It's possible we also lose customers who maybe don't think that we're going eventually go there, but I don't think that's all that likely.